### ISSUE TITLE
Documentation should callout that dotenv.config() needs to be called in seed.ts

### ISSUE THREAD

**Issue by dmzza** [prisma/prisma2#1381]
I deployed to Heroku for the first time and I wanted to use environment variables to configure the path to my datasources. And that also meant I wanted to use a `.env` file. So I followed the documentation: [using environment variables with photonjs](https://github.com/prisma/prisma2/blob/7f8b064e28402740aa29481fbc3aff6a8ef36b25/docs/prisma-schema-file.md#using-environment-variables-with-photonjs)

I knew that `new Photon()` was in `context.ts` so I added `dotenv.config()` to that file, but I didn't think about `new Photon()` in `seed.ts`. So everything appeared to work until I ran `npm run seed` many days later and got this very cryptic error:

```markdown
$ npm run seed

> typescript-graphql@ seed /Users/davidmazza/Sites/pbs-sdk-data
> ts-node prisma/seed

Error: 
Invalid `sansSerifPreset = await photon.presets.upsert()` invocation in /Users/davidmazza/Sites/pbs-sdk-data/prisma/seed.ts:103:48

   99     },
  100   }
  101 }
  102 
→ 103 const sansSerifPreset = await photon.presets.upsert

undefined target="exit" timestamp="2020-01-20T19:24:26.291Z" fields={"message":"1"}
    at PhotonFetcher.<anonymous> (/Users/davidmazza/Sites/pbs-sdk-data/node_modules/@prisma/photon/index.js:65:27)
    at Generator.throw (<anonymous>)
    at rejected (/Users/davidmazza/Sites/pbs-sdk-data/node_modules/@prisma/photon/index.js:6:65)
(node:86429) UnhandledPromiseRejectionWarning: Error: undefined target="exit" timestamp="2020-01-20T19:24:26.291Z" fields={"message":"1"}
    at NodeEngine.engineReady (/Users/davidmazza/Sites/pbs-sdk-data/node_modules/@prisma/photon/runtime/index.js:3545:23)
    at /Users/davidmazza/Sites/pbs-sdk-data/node_modules/@prisma/photon/runtime/index.js:3468:21
(node:86429) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:86429) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

I'm using `preview019`. It would be nice if the error message could have identified the problem, but the error message isn't really the issue here. I think a reminder in [this documentation section](https://github.com/prisma/prisma2/blob/7f8b064e28402740aa29481fbc3aff6a8ef36b25/docs/prisma-schema-file.md#using-environment-variables-with-photonjs) that `dotenv.config()` needs to be called in `seed.ts` would have saved me from this rabbit hole. 

### tl;dr

If you get the error above when running `npm run seed` and you are using a `.env` file, you might have forgotten to add this to `/prisma/seed.ts`

```diff
   import { Photon } from '@prisma/photon'
+  import * as dotenv from 'dotenv'
   
+  dotenv.config() // load the environment variables
   const photon = new Photon()
```

**Comment by Jolg42** [comment]
I think the seed script was removed and the doc is up-to-date so I'm closing this now.

https://github.com/prisma/prisma2/blob/master/docs/prisma-schema-file.md#using-environment-variables-with-prisma-client-js

**Comment by dmzza** [comment]
Is it no longer possible to seed a database with Prisma?

This was never really a feature of the framework anyway, it is just a common technique used start a new database with particular set of data. So it makes sense to document an example of how to do it with prisma, because it’s virtually impossible to live without.

David Mazza

**Comment by Jolg42** [comment]
@dmzza Understood!

So if you have any feedback / feature request about it that issue will be the best place I think https://github.com/prisma/prisma2/issues/9

**Comment by Jolg42** [comment]
And indeed the seed feature was removed as it was not ready. You can create a simple script yourself in the meantime or try https://github.com/prisma/prisma-test-utils/issues/6#issuecomment-565009006
